packages/compass-components/src/components/document-list/element-actions.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import type { TypeCastTypes } from 'hadron-type-checker';
+import { Menu, MenuItem } from '@leafygreen-ui/menu';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import { FontAwesomeIcon } from './font-awesome-icon';
+
+const buttonReset = css({
+  margin: 0,
+  padding: 0,
+  border: 'none',
+  background: 'none',
+});
+
+export const EditActions: React.FunctionComponent<{
+  onRemove?: (() => void) | null;
+  onRevert?: (() => void) | null;
+  editing?: boolean;
+}> = ({ editing, onRemove, onRevert }) => {
+  return (
+    <>
+      {editing &&
+        (onRevert ? (
+          <button
+            className={buttonReset}
+            aria-label="Revert changes"
+            onClick={(evt) => {
+              evt.stopPropagation();
+              onRevert();
+            }}
+          >
+            <FontAwesomeIcon icon="revert"></FontAwesomeIcon>
```

Comment:
Is there a leafygreen icon we could use instead here? Or a plane svg? I'm thinking it might be nice to avoid any FontAwesome imports in `compass-components` if we can avoid it? 

It looks like `Redo` and `ClockWithArrow` from https://www.mongodb.design/component/icon/example/ could potentially work. We might not have them in our current icon version, but maybe we have an explicit svg with a comment about it for now?

Reply:
As I mentioned in the PR description I tried a few of them and even when they are partially matching in design they are too chonky to fit with a 12px font size even in their "small" version (it's the smallest one allowed). Here's how a chevron and plus icon looks like for example:

<img width="163" alt="image" src="https://user-images.githubusercontent.com/5036933/159862475-c2f1ad2c-baba-47d6-a806-beb0c130fcf7.png">

To accomodate for that we would need to change the designs even more somehow and I don't want to do that in this PR, it's an explicit non goal for the project and will extend the timeline significantly.

Worth noting maybe that this `FontAwesomeIcon` is not an import from any FA related library, it's just a local to document-list component that sets some classes to get the icon styles applied. If it's still looks like an issue for you, we might want to consider having all these document-list views in a separate package. It's something we wanted to avoid, but does sound like the only way forward to be able to implement this component following our new guidelines and avoid mentioning FA in compass-components